### PR TITLE
Main frame

### PR DESCRIPTION
**Pull request by Thehashhobo**
commits left out because I forgot to push

### CODE REVIEW

File: src/main/java/entities/Post.java [hunk 1]
```diff
@@ -114,31 +114,28 @@ public LocalDate getDeadline(){
...
      * @param reply the reply to be removed.
-     * @return true if the reply has been removed; false otherwise.
      */
-    public boolean removeReply(int reply){
+    public void removeReply(int reply){
         for(int i = 0; i < repliesIDs.size(); i++){
             if(reply == repliesIDs.get(i)){
                 repliesIDs.remove(i);
-                return true;
+                return;
```
Comment (sushimon) [inline src/main/java/entities/Post.java:136]: Don't need a return statement here if the method returns void

**Review by sushimon — CHANGES_REQUESTED** [review]
Generally looks good, but there are some `void` methods that still have a return statement in them (which can be removed)

**Review by sushimon — DISMISSED** [review]
Previous comments addressed, looks good.

File: src/main/java/filter_post/interface_adapters/FilterPostViewModel.java [hunk 2]
```diff
@@ -6,21 +6,12 @@
...
-    private String[] descriptions;
     private final PropertyChangeSupport observable;
 
     /**
      * Initialize the view model for the filter post use case.
-     * @param titles        A list of titles of all the posts.
-     * @param ids           A list of ids of all the posts.
-     * @param descriptions  A list of descriptions of all the posts.
      */
-    public FilterPostViewModel(String[] titles, int[] ids, String[] descriptions) {
```
[the remark this answers is not in the document]
Comment (Thehashhobo) [reply src/main/java/filter_post/interface_adapters/FilterPostViewModel.java:20]: it was changed bc the constructor now takes no arguments after update

[the remark this answers is not in the document]
Comment (Thehashhobo) [reply src/main/java/filter_post/interface_adapters/FilterPostViewModel.java:20]: btw constructor was changed because they dont need those arguments

**Review by BerryRahul — APPROVED** [review]
looks good

**Review by reganq — APPROVED** [review]
approving

**Review by JohannLeidenfrost — APPROVED** [review]
Lgtm

**Pull request merged**